zipline-loader/src/main/kotlin/app/cash/zipline/loader/topologicalSort.kt

```diff
@@ -0,0 +1,63 @@
+package app.cash.zipline.loader
+
+/**
+ * @param upstreamEdges a function that returns nodes upstream from the input node connected by an edge.
+ *
+ * https://www.interviewcake.com/concept/java/topological-sort
+ */
+fun <T> List<T>.topologicalSort(upstreamEdges: (T) -> Iterable<T>): List<T> {
```

Comment:
Yeah I think we screwed up on the name of this parameter, I think it probably wants to be `sourceToTargets()` ?